docs/CodeTransformation.md

```diff
@@ -110,6 +110,14 @@ interface TransformOptions<OptionType> {
   // names are copied from babel: https://babeljs.io/docs/en/options#caller
   supportsDynamicImport: boolean;
   supportsExportNamespaceFrom: boolean;
+  /**
+   * The value is:
+   * - `false` if Jest runs without Node ESM flag `--experimental-vm-modules`
+   * - `true` if the file extension is defined in [extensionsToTreatAsEsm](https://jestjs.io/docs/next/configuration#extensionstotreatasesm-arraystring)
```

Comment:
hard coding a version will break when we roll new versions. Should use a relative link

Reply:
I hope I put the correct one now.

Reply:
Ping :)